com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/ReadOnlyParametersChecker.java

```diff
@@ -34,27 +35,26 @@ public void check(DeploymentDescriptor descriptor) {
 
     private void checkForCommonParameters(NamedParametersContainer namedParametersContainer, Set<String> readOnlyParameters,
                                           Map<String, Set<String>> commonReadOnlyParameters) {
-        Set<String> commonParameters = Sets.intersection(namedParametersContainer.getParameters()
-                                                                                 .keySet(),
-                                                         readOnlyParameters);
+        Set<String> commonParameters = SetUtils.intersection(namedParametersContainer.getParameters()
```

Comment:
Why this is replaced and formatting is changed?

Reply:
Guava should only be kept as a transitive dependency, there is no need to use it when the Apache commons libraries have simpler implementations for some utility methods, like this one. Otherwise, for the formatting it's a mistake on my part.

Reply:
Good!
I missed that Sets is coming from guava